README.md

```diff
@@ -56,10 +56,10 @@ Basic principles for operation are:
   servers.
 
 JupyterHub also provides a
-[REST API][]
+[REST API](https://jupyterhub.readthedocs.io/en/latest/reference/rest-api.html)
```

Comment:
Using markdown references this way was deliberate. This is not a change we should make.